specification/cognitiveservices/data-plane/VideoAnalysis/videoDescription.tsp

```diff
@@ -0,0 +1,385 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/openapi";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-client-generator-core";
+import "./shared.tsp";
+
+using Azure.Core;
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using TypeSpec.Versioning;
+using OpenAPI;
+using Azure.ClientGenerator.Core;
+
+namespace VideoAnalysis;
+
+@pattern("^[a-z0-9-_]+$")
+@maxLength(32)
+@minLength(1)
+scalar TaskName extends string;
+
+@doc("The ApiVersion query parameter for VideoAnalysisTask API.")
+model VideoAnalysisTaskApiVersionParameter {
+    @doc("The API version to use for this operation.")
+    @query("api-version")
+    @minLength(1)
+    apiVersion?: string;
+}
+
+@doc("Time range considering the offset from 0.0s and lasting a duration.")
+model TimeRange {
+    @doc("Offset in ISO 8601 duration format, i.e., P[n]Y[n]M[n]DT[n]H[n]M[n]S or P[n]W.")
+    offset: string;
```

Comment:
We discourage teams from using ISO 8601 durations as there is poor support for these in high-level languages. Could you recast these as a duration in seconds or minutes represented as an int or float?

Reply:
@mikekistler we are keeping sync with the latest version from [SpeechToText](https://learn.microsoft.com/en-us/rest/api/speechtotext/transcriptions/create?view=rest-speechtotext-v3.2&tabs=HTTP#create-a-transcription-with-language-identification). There is a relationship between Speech and Video, and we wish to make it easier for a user to migrate between these two services seamlessly.

On the review meeting you mentioned that Speech would move away, from this format, can you confirm this?

Reply:
I don't recall making that statement -- are you sure it was me? I do not know what Speech is planning to do on this.